etc/config_ingest_docker.yml

```diff
@@ -27,8 +19,17 @@ xerces: /usr/local/feed/blib/bin/validateCache
 epubcheck: /usr/bin/java -jar /usr/bin/epubcheck
 
 storage_classes:
-  - HTFeed::Storage::LocalPairtree
-  - HTFeed::Storage::VersionedPairtree
+  - class: HTFeed::Storage::LocalPairtree
+    # The directory in which symbolic links will be created to each volume
+    link_dir: /sdr2/obj
+    # The directory into which volumes will be loaded
+    obj_dir: /sdr1/obj
+  - class: HTFeed::Storage::VersionedPairtree
+    # The directory in which symbolic links will be created to each volume
```

Comment:
I probably didn't give enough context to what these were; for `VersionedPairtree` the preferred form for the config is:

```yaml
- Class: HTFeed::Storage::VersionedPairtree
  obj_dir: /htdataden
```

Reply:
(There's probably a corresponding change in `VersionedPairtree` required to use that config key instead of `backup_obj_dir`)

Reply:
`VersionedPairtree` uses `'backup_obj_dir'` in two places: `object_path()` and `stage_path()`. Am I correct in suspecting that `'backup_obj_dir'` as a config key can be considered obsolete (aside from possible legacy use in testing)?

Reply:
Correct, any places that use `backup_obj_dir` should be changed to just use `obj_dir`